.github/actions/README.md

```diff
@@ -47,7 +47,8 @@ Stops sccache server. Use together with `restore-cache`.
 ---
 
 ## `run-e2e-test`
-This action runs a single test from the e2e test suite. It requires a test case, which is the name of the test.
+This action runs a single test from the e2e test suite. It requires a test case, which is the name of the test. It runs
```

Comment:
Would change the last sentence of the comment to:
_It optionally runs the finalization e2e testcase, which is helpful after some e2e tests to double-check nothing is broken._

Reply:
done